tests/app/shared/issue/description/description.component.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { DescriptionComponent } from '../../../../../src/app/shared/issue/description/description.component';
+import { FormBuilder, NgForm } from '@angular/forms';
+import { Phase, PhaseService } from '../../../../../src/app/core/services/phase.service';
+import { Issue } from '../../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../../constants/githubissue.constants';
+import { of } from 'rxjs';
+
+describe('DescriptionComponent', () => {
+  let descriptionComponent: DescriptionComponent;
+  let issueService: any;
+  let phaseService: PhaseService;
+  let formBuilder: FormBuilder;
+  let thisIssue: Issue;
+  let dialog: any;
+  let errorHandlingService: any;
+
+  beforeEach(() => {
+    formBuilder = new FormBuilder();
+    phaseService = new PhaseService(null, null, null, null, null);
+    phaseService.currentPhase = Phase.phaseTeamResponse;
+
+    dialog = jasmine.createSpyObj('MatDialog', ['open']);
+    errorHandlingService = jasmine.createSpyObj('ErrorHandlingService', ['handleError']);
+    issueService = jasmine.createSpyObj('IssueService', ['getLatestIssue', 'updateIssue']);
+
+    descriptionComponent = new DescriptionComponent(issueService, formBuilder, errorHandlingService, dialog, phaseService, null);
+    thisIssue = Issue.createPhaseBugReportingIssue(ISSUE_WITH_EMPTY_DESCRIPTION);
+    descriptionComponent.issue = thisIssue;
+  });
+
+  it('should be initialised with an issueDescriptionForm', () => {
```

Comment:
Let's make the spec description more specific:
`it('should be initialised with a FormGroup instance')`